### DISCUSSION TITLE
Reconstruction error

### DISCUSSION THREAD

**Discussion by hnguyentt** [mlbvn/handson-ml2-vn#60] [Terminology] [answered]
Chương 8 có thuật ngữ `reconstruction error`. Đây là trung bình bình phương khoảng cách giữa dữ liệu gốc và dữ liệu thiết lập lại từ dữ liệu nén bằng PCA.
Mình đề xuất dịch là `độ lỗi tái thiết`.
Mọi người có đề đề xuất nào khác không ạ?

**Comment by rootonchair** [comment]
Cá nhân mình thấy `lỗi phục dựng` nghe hay hơn

cc: @mlbvn/handson-ml2

**Comment by minhduc0711** [comment, marked as answer]
em đang dịch là `khôi phục`, hình như từ này được dùng trong xử lý tín hiệu, phải không nhỉ anh @ngcthuong ?

**Reply by ngcthuong** [reply]
Lỗi khôi phục. Image reconstruction là khôi phục ảnh.
